.github/workflows/release.yaml

```diff
@@ -0,0 +1,514 @@
+name: Build and release
+
+on:
+  push:
+    tags:
+      - '*'
+  schedule:
+    - cron: '0 2 * * *'
+
+jobs:
+  build-linux:
+    name: Build linux binaries
+    runs-on: ubuntu-latest
+    env:
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      S3_HOST: ${{ secrets.S3_HOST }}
+      TARBALL_EXT: tar.xz
+      ARCH: 64
+      DEBIAN_FRONTEND: noninteractive
+      TZ: Asia/Singapore
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          # - image: alpine:3.12
+          #   installCmd: apk update && apk add --no-cache
+          #   toolRequirements: binutils-gold gcc g++ gmp-dev libc-dev libffi-dev make musl-dev ncurses-dev ncurses-static perl tar xz xz-dev automake autoconf coreutils bsd-compat-headers diffutils gzip findutils tree zlib-dev patchelf
+          #   ARTIFACT: "x86_64-linux-alpine"
+          #   DISTRO: Alpine
+          #   ADD_CABAL_ARGS: "--enable-split-sections"
+          # - image: alpine:3.12
+          #   installCmd: apk update && apk add --no-cache
+          #   toolRequirements: binutils-gold gcc g++ gmp-dev libc-dev libffi-dev make musl-dev ncurses-dev ncurses-static perl tar xz xz-dev automake autoconf coreutils bsd-compat-headers diffutils gzip findutils tree zlib-dev patchelf
+          #   ARTIFACT: "x86_64-linux-alpine-static"
+          #   DISTRO: Alpine
+          #   ADD_CABAL_ARGS: "--enable-split-sections --enable-executable-static -f-dynamic"
```

Comment:
Why commneted out?

Reply:
Static alpine bindists were broken for me. There are issues with linking.